Upgrade to 2.26.0-beta.2 fails on API

### Bug description 🐞

When upgrading to `2.26.0-beta.2` the API fails to start because of the following error:

```
Caused by: liquibase.exception.DatabaseException: ERROR: null value in column "branch" of relation "webhook_event" violates not-null constraint
```

### Steps to reproduce

Upgrade to `2.26.0-beta.2` from `2.25.0`

### Expected behavior

_No response_

### Example repository

_No response_

### Anything else?

_No response_

--- a/api/src/main/resources/db/changelog/local/changelog-2.26.0-cli-address.xml
+++ b/api/src/main/resources/db/changelog/local/changelog-2.26.0-cli-address.xml
@@ -18,7 +18,7 @@
             <column name="updated_date" type="datetime" />
             <column name="created_by" type="varchar(128)" />
             <column name="updated_by" type="varchar(128)" />
-            <column name="job_id" type="varchar(36)">
+            <column name="job_id" type="BIGINT">
                 <constraints nullable="false" foreignKeyName="fk_job_address" references="job(id)"/>
             </column>
         </createTable>

